src/main/java/io/snyk/maven/plugins/SnykTest.java

```diff
@@ -262,15 +265,17 @@ private void processVulns(JSONObject responseJson) throws MojoFailureException {
         HashSet<String> vulnIdSet = new HashSet<String>();
 
         JSONArray vulns = (JSONArray)responseJson.get("vulnerabilities");
-        int highestSeverity = SEVERITY_LOW;
+        int highestSeverity = Integer.MIN_VALUE;
 
         Iterator<JSONObject> iterator = vulns.iterator();
         while (iterator.hasNext()) {
             JSONObject vuln = iterator.next();
             vulnIdSet.add((String)vuln.get("id"));
             Integer severityInt = severityMap.get(vuln.get("severity"));
-            if(severityInt != null && severityInt > highestSeverity) {
-                highestSeverity = severityInt;
+            if (!onlyFailFixable || (onlyFailFixable && isIssueFixable(vuln))) {
```

Comment:
This statement can be simplified. Atm second `onlyFailFixable` is always `true`.